Frameworks/UIKit/UILabel.mm

```diff
@@ -558,11 +500,14 @@ - (UIBaselineAdjustment)baselineAdjustment {
 }
 
 /**
- @Status Interoperable
+ @Status Caveat
+ @Notes NSLineBreakByCharWrapping/NSLineBreakByTruncatingHead/NSLineBreakByTruncatingMiddle
+ are not supported. NSLineBreakByCharWrapping is using NSLineBreakByWordWrapping instead.
+ NSLineBreakByTruncatingHead/NSLineBreakByTruncatingMiddle is using NSLineBreakByTruncatingTail instead
```

Comment:
Would help customer understand what was happening.

---
In reply to: [104821652](https://github.com/Microsoft/WinObjC/pull/2178#discussion_r104821652) [](ancestors = 104821652)